indico/modules/events/surveys/models/anonymous_submissions.py

```diff
@@ -0,0 +1,31 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2021 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from indico.core.db import db
+
+
+class SurveyAnonymousSubmission(db.Model):
+    __tablename__ = 'anonymous_submissions'
+    __table_args__ = {'schema': 'event_surveys'}
+
+    #: The ID of the survey
+    survey_id = db.Column(
+        db.Integer,
```

Comment:
you can omit this arg - in case of FKs the type will be inferred from the target column